apis/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -119,6 +120,11 @@ func defaultFeaturesConfig(ddaSpec *DatadogAgentSpec) {
 		apiutils.DefaultStringIfUnset(&ddaSpec.Features.LogCollection.TempStoragePath, defaultLogTempStoragePath)
 	}
 
+	// LiveProcessCollection Feature
```

Comment:
Shouldn't we also define a case for `ddaSpec.Features.LiveProcessCollection == nil` that sets both `Enabled` and `ScrubProcessArguments` to true (according to the new constants defined) ?

Reply:
The answer to this depends on https://github.com/DataDog/datadog-operator/pull/558#discussion_r927528291

Reply:
sorry for the confusion, LiveProcessCollection is not enabled by default but if it is then we set the default value for ScrubProcessArguments